scripts/host-testbench.sh

```diff
@@ -44,6 +44,55 @@ function srcsize() {
   echo "$OUTPUT_SIZE"
 }
 
+function test_comp() {
+    CMD=$1
+    COMP=$2
+    LOGFN=${COMP}_log.txt
+    OUTFN=${COMP}_out.raw
+    echo "=========================================================="
+    echo "test $COMP with ${CMD}"
+    if ${CMD} &> $LOGFN; then
```

Comment:
indentation looks weird at least for me here in firefox - could you check please? First indentation level lines in the function seem to be indented with spaces, second level (within `if` etc.) - with TABs. Is this intended?

Reply:
I removed this part of patch